palace/models/waveportoperator.cpp

```diff
@@ -1066,13 +1113,23 @@ const WavePortData &WavePortOperator::GetPort(int idx) const
   return it->second;
 }
 
+mfem::Array<int> WavePortOperator::GetAttrList() const
+{
+  mfem::Array<int> attr_list;
+  for (const auto &[idx, data] : ports)
+  {
+    attr_list.Append(data.GetAttrList());
+  }
+  return attr_list;
+}
+
 void WavePortOperator::Initialize(double omega)
 {
   bool init = false, first = true;
   for (const auto &[idx, data] : ports)
   {
-    init = init || (data.GetOperatingFrequency() != omega);
-    first = first && (data.GetOperatingFrequency() == 0.0);
+    init = init || (data.omega0 != omega);
+    first = first && (data.omega0 == 0.0);
```

Comment:
This changes meaning to bitwise or instead of logical, which as far as I know is bad practice (from C, though). See https://stackoverflow.com/questions/24542/using-bitwise-operators-for-booleans-in-c.